shared/src/main/scala/uk/gov/homeoffice/drt/arrivals/Arrival.scala

```diff
@@ -132,16 +139,17 @@ case class Arrival(Operator: Option[Operator],
     (minutesToDisembark * oneMinuteInMillis).toLong
   }
 
-  val excludeTransferPax: Option[Int] => Option[Int] = maybePax =>
-    maybePax.map(_ - TranPax.getOrElse(0)) match {
-      case Some(nonNegative) if nonNegative >= 0 => Some(nonNegative)
-      case Some(_) => Some(0)
-      case None => None
+  val excludeTransferPax: Option[Passengers] => Option[Passengers] = maybePax =>
```

Comment:
This function is redundant now since there's a `getPcpPax` function on `Passengers`. We just need to move the logic to deal with negative pax in there

Reply:
We don't need this function - it's duplicating logic from `Passengers.getPcpPax`